lineman/app/components/group_page/group_page.haml

```diff
@@ -1,32 +1,37 @@
 .loading-wrapper.container.main-container
   %loading{ng-if: '!groupPage.group'}
   %main.group-page{ng-if: 'groupPage.group'}
-    .group-page__cover{style:'background-image:url({{groupPage.group.coverUrl()}})'}
-    %section.group-page__header{aria-label: "{{ 'group_page.header.aria_label' | translate }}"}
-      .group-page__logo{style:'background-image:url({{groupPage.group.logoUrl()}})', alt: "{{ 'group_page.group_logo' | translate }}"}
-      .group-page__name-and-actions
-        .group-page__name
-          %h1.lmo-h1
-            %a{ng_show: 'groupPage.group.parentId', href: '/g/{{groupPage.group.parent().key}}'}
-              {{groupPage.group.parentName()}}
-            %span{ng_show: 'groupPage.group.parentId'}> \-
-            {{groupPage.group.name}}
-        %group_privacy_dropdown{group: 'groupPage.group'}
-        .group-page__member-actions.lmo-btn-group.lmo-btn-group-right{ng-if: 'groupPage.isMember()'}
-          .lmo-btn-wrapper
-            %group_actions_dropdown{group: 'groupPage.group'}
-          .lmo-btn-wrapper
-            %notification_volume_dropdown{translate-root: 'group_page', group: 'groupPage.group'}
-        %join_group_button{group: 'groupPage.group'}
-        .clearfix
-    .row
-      .col-sm-8
-        %section.group-page__description{aria-labelledby: 'description-card-title'}
-          %h2#description-card-title.lmo-card-heading{ translate: 'group_page.description'}
-          .group-page__description-placeholder.lmo-placeholder.lmo-hint-text{translate: 'group_page.description_placeholder', ng-if: 'groupPage.showDescriptionPlaceholder()' }
-          .group-page__description-text.lmo-last.lmo-markdown-wrapper{ marked: 'groupPage.group.description' }
-        %discussions_card{group: 'groupPage.group'}
-      .col-sm-4
-        %membership_requests_card{group: 'groupPage.group'}
-        %members_card{group: 'groupPage.group'}
-        %subgroups_card{group: 'groupPage.group'}
+    .group-page__cover{ng-style:'groupPage.coverStyle()'}
+      %a.group-page__upload-photo{href: '', ng-click: 'groupPage.openUploadCoverForm()', ng-show: 'groupPage.canUploadPhotos()'}
+        %i.fa.fa-lg.fa-camera
+    %main.container.main-container
```

Comment:
FYI - We had scratch our heads a little bit to remove the double %main element already.. this is probably what is currently conflicting. Can you please ensure we don't have two main elements when the merge happens